producer.go

```diff
@@ -355,6 +444,30 @@ func getDefaultProduceOpts() ProduceOpts {
 
 // Producer.Produce - produces a message into a station. message is of type []byte/protoreflect.ProtoMessage in case it is a schema validated station
 func (p *Producer) Produce(message any, opts ...ProduceOpt) error {
+	if p.isMultiStationProducer {
+		return p.produceToMultiStation(message, opts...)
+	}
+
+	return p.produceToSingleStation(message, opts...)
+}
+
+func (p *Producer) produceToMultiStation(message any, opts ...ProduceOpt) error {
+	stations := p.stationName.([]string)
+	var wg sync.WaitGroup
+
+	for _, station := range stations {
+		wg.Add(1)
+		go func(sn string) {
```

Comment:
when producing many messages too many go routines will be created and the program will panic,
change the produce to work consecutively and return an error to the user if a produce doesn't succeed.